tests/test_integration_scenarios.py

```
@@ -406,21 +411,25 @@ def test_zero_check_interval_validation(self, mock_config, mock_state, mock_logg
         from gengowatcher.watcher import GengoWatcher
 
         mock_config.get.side_effect = lambda s, k, **kw: {
-            ("Watcher", "check_interval"): 0
-        }.get((s, k), kw.get("fallback", 60))
+            ("Watcher", "check_interval"): 0,
+            ("Logging", "log_all_entries_enabled"): False,
+        }.get((s, k), kw.get("fallback"))
 
         watcher = GengoWatcher(mock_config, mock_state, mock_logger)
```

Ruff (0.14.14)

[error] Local variable `watcher` is assigned to but never used

Remove assignment to unused variable `watcher`

(F841)